back/app/controllers/web_api/v1/projects_controller.rb

```diff
@@ -145,6 +169,9 @@ def index_for_topics
   end
 
   def index_for_admin
+    # In this endpoint, we always want to include unlisted projects.
+    policy_context[:apply_listed_scope] = true
+
```

Comment:
Do we actually need this? Won't unlisted projects always be returned if we don't specify a scope?